web/controllers/v1/bundle_dynamic_config_controller.ex

```diff
@@ -76,27 +76,34 @@ defmodule Cog.V1.BundleDynamicConfigController do
       nil ->
         not_found(conn, "Bundle #{bundle_id} not found")
       bundle ->
-        case Bundles.create_dynamic_config_for_bundle(bundle, params) do
-          {:ok, dyn_config} ->
-            location = case Map.get(params, "layer") do
-                         "base" ->
-                           Helpers.bundle_dynamic_config_path(conn, :show_layer,
-                                                              dyn_config.bundle_id,
-                                                              dyn_config.layer, %{})
-                         _ ->
-                           Helpers.bundle_dynamic_config_path(conn, :show_layer,
-                                                              dyn_config.bundle_id,
-                                                              dyn_config.layer,
-                                                              dyn_config.name, %{})
-                       end
-            conn
-            |> put_status(:created)
-            |> put_resp_header("location", location)
-            |> render("show.json", dynamic_config: dyn_config)
-          {:error, changeset} ->
+        try do
+          case Bundles.create_dynamic_config_for_bundle!(bundle, params) do
```

Comment:
Not a huge deal but it is a little weird to suffix with `!` when the function itself doesn't raise.